Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook

```diff
@@ -3516,7 +3474,2305 @@
+          }
+        ],
+        "exportParameters": true
+      },
+      "conditionalVisibilities": [
+        {
+          "parameterName": "SelectedTab",
+          "comparison": "isEqualTo",
+          "value": "Performance"
+        },
+        {
+          "parameterName": "param_check_error_processutilization",
+          "comparison": "isEqualTo",
+          "value": "0"
+        }
+      ],
+      "name": "PerformanceTabGroup"
+    },
+    {
+      "type": 12,
+      "content": {
+        "version": "NotebookGroup/1.0",
+        "groupType": "editable",
+        "items": [
+          {
+            "type": 12,
+            "content": {
+              "version": "NotebookGroup/1.0",
+              "groupType": "editable",
+              "items": [
+                {
+                  "type": 3,
+                  "content": {
+                    "version": "KqlItem/1.0",
+                    "query": "let appServerList = dynamic([{CpuMemApplicationServer}]);\r\nlet baseQuery = SapNetweaver_SMON_CL\r\n| where timestamp_t between (todatetime({CpuMemTimeRange:start}) .. todatetime({CpuMemTimeRange:end}))\r\n| where SID_s == '{CpuMemSID}'\r\n| project USR_TOTAL_d, SYS_TOTAL_d, IDLE_TOTAL_d, SID_s, hostname_s, instanceNr_s, timestamp_t,appServerList\r\n| extend instanceNr_s = iff(strlen(instanceNr_s) == 1, strcat(\"0\", instanceNr_s), instanceNr_s)\r\n| extend appServer = strcat(hostname_s, \"_\", SID_s, \"_\", instanceNr_s)\r\n| extend TOTAL_CPU = (100 - IDLE_TOTAL_d)\r\n| where array_length(appServerList) == 0 or strcat(hostname_s, \"_\", instanceNr_s) in (appServerList);\r\nlet timespanCount = baseQuery \r\n| summarize RecordCount=count()\r\n| extend Interval = iff(RecordCount <= 10, timespan(1m), timespan(1m))\r\n| extend Interval = iff(RecordCount > 10 and RecordCount < 50, timespan(5m), Interval)\r\n| extend Interval = iff(RecordCount >= 50 and RecordCount < 100, timespan(10m), Interval)\r\n| extend Interval = iff(RecordCount >= 100 and RecordCount < 150, timespan(15m), Interval)\r\n| extend Interval = iff(RecordCount >= 150, timespan(30m), Interval)\r\n| project Interval;\r\nlet timespanInterval = toscalar(timespanCount);\r\nbaseQuery\r\n| summarize Count = count() by bin(timestamp_t, timespanInterval),USR_TOTAL_d, SYS_TOTAL_d, IDLE_TOTAL_d, TOTAL_CPU, timestamp_t",
+                    "size": 0,
+                    "aggregation": 3,
+                    "title": "CPU Utilization - User / System",
+                    "queryType": 0,
+                    "resourceType": "microsoft.operationalinsights/workspaces",
+                    "visualization": "areachart",
+                    "chartSettings": {
+                      "xAxis": "timestamp_t",
+                      "yAxis": [
+                        "USR_TOTAL_d",
+                        "SYS_TOTAL_d"
+                      ],
+                      "group": null,
+                      "createOtherGroup": 0,
+                      "seriesLabelSettings": [
```

Comment:
i think we have some duplicate labels here, 2x each for CPU User Utilization and CPU System Utilization, but one set of labels actually specified color.